instrumentation/jdbc/src/main/java/io/opentelemetry/auto/instrumentation/jdbc/PreparedStatementInstrumentation.java

```diff
@@ -75,28 +76,31 @@ public PreparedStatementInstrumentation() {
     public static void onEnter(
         @Advice.This final PreparedStatement statement,
         @Advice.Local("otelSpan") Span span,
-        @Advice.Local("otelScope") Scope scope) {
+        @Advice.Local("otelScope") Scope scope,
+        @Advice.Local("otelCallDepth") Depth callDepth) {
 
-      span = TRACER.startSpan(statement, JDBCMaps.preparedStatements.get(statement));
-      if (span != null) {
-        scope = TRACER.withSpan(span);
+      callDepth = TRACER.getCallDepth();
+      if (callDepth.getAndIncrement() == 0) {
+        span = TRACER.startSpan(statement, JDBCMaps.preparedStatements.get(statement));
```

Comment:
move query lookup to overloaded `JdbcTracer.startSpan(Statement)`, since that will be useful for manual instrumentation also?

```suggestion
        span = TRACER.startSpan(statement);
```